Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionDataRecord.mm

```diff
@@ -206,7 +220,6 @@
 
     auto *backgroundScriptTwo = Util::constructScript(@[
         @"const data = { 'string': 'string', 'number': 1, 'boolean': true, 'dictionary': {'key': 'value'}, 'array': [1, true, 'string'] }",
-        @"await browser?.storage?.session?.set(data)",
```

Comment:
thought the issue could've been with session storage not being deleted, since it's handled differently than `local` and `sync`. however, the test still fails with this removed. will add back and will try to figure out why this test is still failing